### PR TITLE
Fixed #76 - How to use this book content

### LINKED ISSUE #76: How to Use This Book content needed
We need to provide guidance on how to download and execute notebooks. We should probably also consider adding a callout in every notebook saying "you can browse the rendered notebook, but if you want to interact with the book contents you'll need to download and execute. Here's how..."

### LINKED ISSUE #110: Issue on page /preamble/how-to-use.html
State clearly who this is for. I am a fan of 'preamble'; 'introduction' or 'getting started' are clearer to me.

### PR DESCRIPTION
**Pull request by clyne**
Fixes #76 
Fixes #110

This PR adds "how to use this book" content to Pythia Foundations

### CODE REVIEW

File: preamble/how-to-use.md [hunk 1]
```diff
@@ -1,5 +1,111 @@
...
+Foundations applies broadly, and may be useful to scientists, or
+aspiring scientists, working in any number of disciplines. Pythia
+Foundations makes very few assumptions about the experience level
+of the reader other than having a background in math or science,
+and being comfortable using a computer, including the command line
+terminal (i.e. the [Unix shell](https://en.wikipedia.org/wiki/Unix_shell)).
+Prior programming experience is not required.
+
+Lastly, we note that Pythia Foundations covers two topics that are
+not technically part of the SPE, but that we believe are essential
```
[the remark this answers is not in the document]
Comment (mgrover1) [reply preamble/how-to-use.md:19]: I would agree here - I think many believe Jupyter Notebooks to be a key part of the scientific python ecosystem...

**Review by mgrover1 — COMMENTED** [review]
Just a small comment - other than that, this looks fantastic!

File: preamble/how-to-use.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (clyne) [reply preamble/how-to-use.md]: See if it makes more sense now. Thanks

**Review by mgrover1 — APPROVED** [review]
Looks good to go - thanks @clyne

**Review by dopplershift — APPROVED** [review]

File: preamble/how-to-use.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dopplershift) [reply preamble/how-to-use.md]: Sounds better. Thanks!

**Pull request merged**